DOCS-2722 Context Menu Link Guides for Dashboards (#11730)
DataDog/documentation: content/en/dashboards/widgets/_index.md

## Overview

> The widgets are the building blocks of your Dashboards. There are essentialy three different categories: 
> 
> * **Generic** widgets, meant to graph data from Datadog Products. 
> * **Summary** widgets, meant to display synthetised monitoring information. 
> * **Decoration** widgets, meant to visually structure and annotate Dashboards, for better readability.

Reviewer: ```suggestion
The widgets are the building blocks for your dashboards and are categorized as: 

* **Generic** widgets to graph data from Datadog products 
* **Summary** widgets to display Synthetic monitoring information 
* **Decoration** widgets to visually structure and annotate dashboards
```

Additional options are available for [timeseries widgets][1].

## Custom links

> Refer to our [guide][6] to customize interactions with data inline generic widgets. 

Reviewer: ```suggestion
To customize interactions with data inline generic widgets, see [Custom Links][6]. 
```

[3]: /notebooks/
[4]: /tracing/visualization/service/
[5]: /tracing/visualization/resource/

> [6]: /dashboards/guide/custom_links/

Reviewer: This link is breaking our link checker. 

```suggestion
[6]: 
```
